src/testhost.x86/DefaultEngineInvoker.cs

```diff
@@ -62,14 +60,19 @@ public void Invoke(IDictionary<string, string> argsDictionary)
                 // Attach to exit of parent process
                 var parentProcessId = CommandLineArgumentsHelper.GetIntArgFromDict(argsDictionary, ParentProcessIdArgument);
                 EqtTrace.Info("DefaultEngineInvoker: Monitoring parent process with id: '{0}'", parentProcessId);
-                var processHelper = new ProcessHelper();
-                processHelper.SetExitCallback(
-                    parentProcessId,
-                    () =>
-                        {
-                            EqtTrace.Info("DefaultEngineInvoker: ParentProcess '{0}' Exited.", parentProcessId);
-                            Environment.Exit(1);
-                        });
+
+                // In remote scenario we cannot monitor parent process, so we expect user to pass parentProcessId as -1
```

Comment:
The responsibility to clean up test runner lies with it's RuntimeProvider, but if vstest.console is dead then runtime provider won't get a chance to clean up, the best we can achieve here is safely exit the UWP app.